Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -160,9 +173,11 @@ const KeyboardAvoidingView = React.createClass({
 
       case 'position':
         const positionStyle = {bottom: this.state.bottom};
+        const { contentContainerStyle } = this.props;
+
         return (
           <View ref={viewRef} style={style} onLayout={this.onLayout} {...props}>
-            <View style={positionStyle}>
+            <View style={[positionStyle, contentContainerStyle]}>
```

Comment:
@javache Isn't it better to allow overriding position? If someone doesn't want to override position, he won't pass a prop.

Use case: say there's a overlayed view, in which case you only want to adjust the bottom in the overlayed view, in the underlying view, you can disable the behaviour by passing `bottom: 0`